### PR TITLE
fix readme files

### PR DESCRIPTION
**Pull request by vadiminc**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,8 +97,8 @@ To run you will use the "shares" command
 - operator-ids (oids) = Comma-separated list of operator IDs. The amount must be 3f+1 compatible.
 - operator-keys (oks) = Comma-separated list of operator keys (same sequence as operator ids). The amount must be 3f+1 compatible.
 - output-folder (of) = Target folder path to output the key shares file
-- owner-address (oa) = The cluster owner address
-- owner-nonce (on) = The nonce of the owner within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool
+- owner-address (oa) = The cluster owner address (in the SSV contract)
+- owner-nonce (on) = The validator registration nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.
```
Comment (meshin-dev) [inline README.md:101]: If you use dots in the end of all sentences - use in all, not only few.
If not - then delete it from everywhere

File: src/commands/actions/arguments/owner-nonce.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default {
   options: {
     type: Number,
     required: true,
-    help: 'The validator owner nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.'
+    help: 'The validator registration nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.'
```
Comment (meshin-dev) [inline src/commands/actions/arguments/owner-nonce.ts:7]: The same is for all the `help` options, some of them I see without dots and some - with

File: src/commands/actions/arguments/owner-address.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ export default {
   options: {
     type: String,
     required: true,
-    help: 'The cluster owner address'
+    help: 'The cluster owner address (in the SSV contract)'
```
Comment (meshin-dev) [inline src/commands/actions/arguments/owner-address.ts:7]: here

**Review by meshin-dev — APPROVED** [review]
Comments only about dots in sentences everywhere (in code + readme).
I didn't check if text matches PRD, but assuming that yes.
Approved

**Review by andrew-blox — APPROVED** [review]

**Review by Tomblox1 — APPROVED** [review]

**Pull request merged**